_content/news/_posts/2019-01-14-hervorming.markdown

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: "Hervorming jeugdcompetitie"
+date: 2019-01-07 10:00:00
+description: Volgend seizoen gaat de jeugdcompetitie er heel anders uitzien
+cover: /news/img/2018-12-28-lizards-grid-rsz.jpg
+permalink: /news/2019-01-14-hervorming/
+---
+
+Op de Algemene Vergadering van basketbal vlaanderen werd in november 2018 de hervorming van de jeugdcompetitie goedgekeurd.
```

Comment:
Op de Algemene Vergadering van VBL (basketbalbond) werd in november 2018 de hervorming van de jeugdcompetitie goedgekeurd.